pkg/drivers/qemu/qemu.go

```diff
@@ -273,19 +272,19 @@ func parsePortRange(rawPortRange string) (int, int, error) {
 
 	minPort, err := strconv.Atoi(portRange[0])
 	if err != nil {
-		return 0, 0, fmt.Errorf("invalid port range")
+		return 0, 0, errors.Wrap(err, "Invalid port range")
 	}
 	maxPort, err := strconv.Atoi(portRange[1])
```

Comment:
We don't have any check verifying that `portRange` has a length of two, it's probably a small chance but would probably be worth checking the length and failing with `malformed port range`.

Reply:
Good call, fixed.